sys/linux/ion.txt

```diff
@@ -6,6 +6,7 @@
 # TODO: ION_IOC_HEAP_QUERY is detected for 4.10 upstream, but not covered in this description.
 
 include <asm/ioctl.h>
+include <uapi/linux/fcntl.h>
```

Comment:
`uapi` should be the first on syz-extract lookup path. Does it work without `uapi/`? If yes, remote it.